src/oauth2c.erl

```diff
@@ -89,6 +93,13 @@
 
 %%% API ========================================================================
 
+start() ->
```

Comment:
oauth2_client is (before this PR) a library application without any processes of its own. Since this adds a process, it should also add a supervisor and provide a start_link function which starts this supervisor

Reply:
I have added a supervisor removed the start function from oauth2c.